app/internal/page/mock.go

```diff
@@ -0,0 +1,197 @@
+package page
```

Comment:
sorry missed the filename before, this should be `mock_test.go` so it doesn't get compiled into normal code

Reply:
I'd initially tried this but as soon as I postfix a `_test` on it the exported types and vars become unavailable in other packages. 

If we don't want this to be compiled is there some way of changing the build command to exclude directories/files?